### ISSUE TITLE
Item Issue - Juno's Shadow

### ISSUE THREAD

**Issue by Hacheteroni** [elysium-project/server#2358]
Once you've read this message, delete it and write your report below.

Write your report here

Item - Juno's Shadow
Item ID - 17061
Stats - 10 Stamina / 5 Intellect / 15 Shadow Resistance. Requires Level 53.
Drops in - Stratholme both sides - TRASH MOBS.

Does the above item actually exist in the server? Cant link from atlas loot and there are NONE ever on AH. This will be a sought after item for warlocks in AQ40 for twin emp tanking. Can someone confirm to me if it is or isnt. If it isnt, lets get it coded in boys. :)

#Varbane.

**Comment by Sabre142** [comment]
Yes, it should be in according to this. http://web.archive.org/web/20060510042440/http://wow.allakhazam.com:80/db/mob.html?wmob=10405

**Comment by Hacheteroni** [comment]
Cool. So can you confirm what patch? Maybe it should be here now.

**Comment by Sabre142** [comment]
Patch 1.10. https://github.com/elysium-project/database/blob/master/progression/1.9%20%E2%86%92%201.10%20Storms%20of%20Azeroth.sql
